fvtest/porttest/si.cpp

```diff
@@ -2343,3 +2343,37 @@ TEST(PortSysinfoTest, sysinfo_cgroup_get_memlimit)
 	reportTestExit(OMRPORTLIB, testName);
 	return;
 }
+
+/**
+ * Test omrsysinfo_get_processor_description.
+ */
+TEST(PortSysinfoTest, sysinfo_test_sysinfo_get_processor_description)
+{
+	OMRPORT_ACCESS_FROM_OMRPORT(portTestEnv->getPortLibrary());
+	const char *testName = "omrsysinfo_get_processor_description";
+	int32_t rc = 0;
+	int i = 0;
```

Comment:
This is C++, don't declare locals up front. This `i` local can be declared in the for loop below. The `rc` should be beside the declaration for `desc`